indigo/distribution.yaml

```diff
@@ -780,6 +780,14 @@ repositories:
       url: https://github.com/kejriwalnishant1990/bowpmap_ros.git
       version: master
     status: maintained
+  bowpmap_ros-release:
```

Comment:
I have kept it bowpmap_ros only. No -release tag. I think there is some error in my package building and i am rectifying it and once i do it i will let you know.